django/db/backends/postgresql/compiler.py

```diff
@@ -4,7 +4,9 @@
     SQLDeleteCompiler,
 )
 from django.db.models.sql.compiler import SQLInsertCompiler as BaseSQLInsertCompiler
-from django.db.models.sql.compiler import SQLUpdateCompiler
+from django.db.models.sql.compiler import (
+    SQLUpdateCompiler,
+)
```

Comment:
I'm not sure this is a desired change, was `isort` the one that changed this?

Reply:
https://github.com/PyCQA/isort/releases/tag/6.0.0
I think `Ensure that split_on_trailing_comma works with as imports` is suspicious (https://github.com/PyCQA/isort/pull/2340)

Reply:
I guess we could also pin the `isort` version for the being to avoid back and forth if this is fixed in `6.0.1`. First versions of software usually have rough edges.

Reply:
I have commented in the change mentioning this undesired reformat, for now I'd be inclined to ping `isort` to unblock CI. Working on a PR now!

Reply:
I created an issue: https://github.com/PyCQA/isort/issues/2352

Reply:
@nessita it is. Please check my comment on the issue created on isort.